app.json

```diff
@@ -222,6 +222,14 @@
     },
     "VOUCHER_INTERNATIONAL_SCHOOL_KEY": {
       "required": false
+    },
+    "NEW_COURSES_FORM_GUID": {
+      "description": "Form guid over hub spot for new courses email subscription form.",
+      "value": "ad5d54e5-5ca9-4255-9c17-fa222e0a9b82"
+    },
+    "HUBSPOT_PORTAL_ID": {
+      "description": "Hub spot portal id.",
+      "value": "4994459"
```

Comment:
@shaidar I have added these values in `app.json` for heroku. let me know anything else need to consider ?

Reply:
Can you namespace the first one with `HUBSPOT_` as well and specify both as `"required": false`?